Makefile.am

```diff
@@ -135,7 +135,7 @@ install-test-requires: install-buildrequires install-requires
 		dnf install bzip2 cppcheck gnome-icon-theme gnome-icon-theme-symbolic \
 		    lorax mock parallel rpm-ostree virt-install pykickstart spin-kickstarts  \
 		    python3-rpmfluff python3-mock python3-pocketlint python3-nose-testconfig \
-		    python3-sphinx_rtd_theme libvirt-python3 python3-lxml
+		    python3-sphinx_rtd_theme libvirt-python3 python3-lxml rpm-build
```

Comment:
This looks like a bug in rpmfluff:
https://bugzilla.redhat.com/show_bug.cgi?id=1259500